src/inc/palclr.h

```diff
@@ -484,7 +484,14 @@
 //  and it provides __declspec(selectany) to instruct the linker to merge
 //  duplicate external const static data copies into one.
 //  
+#if defined(__clang__)
+#define SELECTANY extern
```

Comment:
What is the reason for this change? I believe Clang supports the __declspec(selectany).